lms/urls.py

```diff
@@ -84,6 +84,9 @@
     # Course content API
     url(r'^api/course_structure/', include('course_structure_api.urls', namespace='course_structure_api')),
 
+    # Course API
+    url(r'^api/courses/', include('course_api.urls')),
```

Comment:
Update: Looks like plural is the way to go which is good. Perhaps the course_api folder should be called courses_api instead?